File: InterestingQ/Alternate.cpp
#include<iostream>
#include<cstring>

using namespace std;

int main()
{
    string str;

    cout<<"Enter your Name:";
    getline(cin,str);

    int i=0;
    int j=0;
    int len=0;

    while(str[len]!='\0')
    {
        len++;
    }
    for(i=0;i<len;i++)
     {
        if(j%2==1)
        {
            if(str[i]>65 && str[i]<90)
            str[i]+=32;
        }
        else
        {
            if(str[i]>95 && str[i]<122)
            str[i]-=32;
        }
        j++;
    }

    cout<<"After Converting : "<<str<<endl;
}

File: Alternate.java
import java.util.Scanner;
public class Alternate{

 public static void main(String[] args) {
  Scanner cs=new Scanner(System.in);
  String str1;
  System.out.println("Enter your String:");
  str1=cs.nextLine();
  char[] str=str1.toCharArray();
  int j=0;
  
  for(int i=0;i<str.length;i++)
     {
         if(j%2==1){
         
   if(str[i]>64 && str[i]<91 )
    str[i]=(char)((int)str[i]+32);
         }
         else{
             if(str[i]>96 && str[i]<123 )
              str[i]=(char)((int)str[i]-32);
         }
         if(str[i]==' ')
         {
           continue;
         }
         j++;
     }
  System.out.print("After Converting String is : ");
  for(int i=0;i<str.length;i++)
  System.out.print(str[i]);
  cs.close();
 }
}